frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1119,6 +1129,42 @@ pub mod pallet {
 			<QueuedSolution<T>>::put(solution);
 			Ok(())
 		}
+
+		/// Rotates the round which results in setting the current phase to
+		/// `Phase::Off` and removes all the current solutions.
+		///
+		/// The dispatch origin for this call must be `ForceOrigin`.
+		#[pallet::call_index(5)]
+		#[pallet::weight(({
+			T::WeightInfo::force_rotate_round()
+		}, DispatchClass::Operational))]
+		pub fn force_rotate_round(
+			origin: OriginFor<T>,
+			current_phase: Phase<T::BlockNumber>,
+		) -> DispatchResult {
+			T::ForceOrigin::ensure_origin(origin)?;
+			ensure!(current_phase == Self::current_phase(), <Error<T>>::CallNotAllowed);
+
+			Self::do_force_rotate_round();
+
+			Ok(())
+		}
+
+		/// Indicates that by the end of the current session the phase will be
+		/// forced to the specified phase.
+		///
+		/// The dispatch origin for this call must be root.
+		#[pallet::call_index(6)]
+		#[pallet::weight((T::DbWeight::get().writes(1), DispatchClass::Operational))]
+		pub fn force_start_phase(
+			origin: OriginFor<T>,
+			phase: Phase<T::BlockNumber>,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			<ForcePhase<T>>::put(phase);
```

Comment:
Fail if phase isn't one of supported phases to be forced? (Signed and Emergency)